test/README.md

```diff
@@ -0,0 +1,14 @@
+# Tests
+
+This repository contains amber scripts to test changes and protect from
+regressions.
+
+## Amber
+
+The `run_amber.py` script runs all amber scripts inside the `amber` directory.
+This needs a GPU with a working Vulkan driver and amber inside the `PATH`.
```

Comment:
So this doesn't pick up amdvlk from the build directory and point `VK_ICD_FILENAMES` to it?